src/platform/site-wide/cta-widget/index.js

```diff
@@ -79,8 +79,8 @@ export class CallToActionWidget extends React.Component {
     }
 
     if (this.isAccessible()) {
-      const { appId, useSSOe } = this.props;
-      const { url, redirect } = toolUrl(appId, useSSOe);
+      const { appId, authenticatedWithSSOe } = this.props;
+      const { url, redirect } = toolUrl(appId, authenticatedWithSSOe);
```

Comment:
@hinzed1127 I think this starts going down the path of whether we also want to update the MHV/eBenefit URLs based on the user's state rather than the feature flag....

Reply:
that's a good point. considering those are more encapsulated by their own functions, I don't think it actually adds that much more to this PR